src/usr/local/www/services_unbound.php

```diff
@@ -317,7 +317,9 @@ function build_if_list($selectedifs) {
 	'DNS Query Forwarding',
 	'Enable Forwarding Mode',
 	$pconfig['forwarding']
-));
+))->setHelp(sprintf('If this option is set, DNS queries will be forwarded to the upstream DNS servers defined under'.
+					' %sSystem: General Setup%s or those obtained via DHCP/PPP on WAN'.
```

Comment:
We've been using the format "Menu Name > Menu Selection" in documentation so we should use that in the GUI as well.